packages/telemetry/src/index.ts

```diff
@@ -112,39 +143,76 @@ export class Telemetry implements TelemetryApi {
     name,
     attachType = 'child',
     active = false,
+    parentSpan,
     opts = {},
+    key,
+    isVerbose = false,
```

Comment:
did you consider creating a wrapper called `createVerboseSpan` which would remove the need to define isVerbose throughout the code?

```
const span = startVerboseSpan(spanOpts)

const startVerboseSpan = (opts) => {
  if (!this.verbose) return undefined

   return startSpan(opts)
}
```

Reply:
I think how we have it is fine for now, a single command with options... In the future we should enhance verbose to allow targeted span creation (the real question there is how to filter and ensure parent spans are created).

Reply:
My thought is it would remove the need to pass `isVerbose` throughout the code-base and need to export the isVerbose boolean

Reply:
I mean we could remove that need just by setting it in the proxy, that `isVerbose` export is just for filtering verbosity on an off within the proxy, not the entire code base. 🤷‍♂️ The better way to do that would be a better filtering mechanism like we have with debugging, but for now our cheap 'isVerbose' works well enough. (imo)